src/goInstallTools.ts

```diff
@@ -355,11 +355,22 @@ export async function promptForUpdatingTool(toolName: string) {
 	}
 	const goVersion = await getGoVersion();
 	const updateMsg = `Your version of ${tool.name} appears to be out of date. Please update for an improved experience.`;
-	vscode.window.showInformationMessage(updateMsg, 'Update').then((selected) => {
+	var choices: string[] = ['Update'];
+	if (toolName == `gopls`) {
+		choices.push('Release Note');  // TODO(hyangah): pass more info such as version, release note location.
+	}
+	vscode.window.showInformationMessage(updateMsg, ...choices).then((selected) => {
 		switch (selected) {
 			case 'Update':
 				installTools([tool], goVersion);
 				break;
+			case 'Release Note':
+				if (toolName == `gopls`) {
+					vscode.commands.executeCommand('vscode.open', vscode.Uri.parse('https://github.com/golang/go/issues/33030#issuecomment-510151934'));			
+				} else {					
```

Comment:
A few lines above, we add the `Release Note` option only when the tool is `gopls`. Therefore, this else block will never be reached

Reply:
you're right. removed.